lamindb/curators/__init__.py

```diff
@@ -1034,10 +1039,15 @@ def __init__(
         if exclude is None:
             exclude = {}
         self._exclude = exclude
-        self._dataset = mdata
         self._organism = organism
         self._var_fields = var_index
         self._columns_field = var_index  # this is for consistency with BaseCatCurator
+        if isinstance(mdata, Artifact):
+            self._artifact = mdata
+            self._dataset = mdata.load()
```

Comment:
This is a bit dangerous given this thing might be huge and it might lead to an unexpected download, but I don't see another way for now.

Reply:
I agree.